amxmodx/CPlugin.cpp

```diff
@@ -154,6 +154,11 @@ int CPluginMngr::loadPluginsFromFile(const char* filename, bool warn)
 				*get_amxaddr(plugin->getAMX(), addr) = gpGlobals->maxClients;
 			}
 
+			if (amx_FindPubVar(plugin->getAMX(), "MapName", &addr) != AMX_ERR_NOTFOUND)
+			{
+				set_amxstring(plugin->getAMX(), addr, STRING(gpGlobals->mapname), PLATFORM_MAX_PATH);
```

Comment:
`PLATFORM_MAX_PATH` is not correct anymore.

Reply:
It should be changed to `64 - 1` according to `MAX_MAPNAME_LENGTH`. Is there any named constant that can be used here?

Reply:
I did not find one. And I didn't want to create a new one.

Reply:
@KaidoRen but anyway you should change it to `64 - 1`.